pkg/plugins/policies/meshfaultinjection/plugin/v1alpha1/plugin.go

```diff
@@ -71,9 +71,6 @@ func applyToInbounds(
 	for _, inbound := range proxy.Dataplane.Spec.GetNetworking().GetInbound() {
 		iface := proxy.Dataplane.Spec.Networking.ToInboundInterface(inbound)
 		protocol := core_mesh.ParseProtocol(inbound.GetProtocol())
-		if _, exists := proxy.Policies.FaultInjections[iface]; exists {
```

Comment:
why was this here in the first place @lukidzi ? why do we not need it now @lahabana ?